crates/miden-objects/src/account/storage/map/storage_map_key.rs

```diff
@@ -0,0 +1,101 @@
+use core::fmt;
+
+use miden_crypto::Word;
+use miden_crypto::word::LexicographicWord;
+
+use super::{
+    ByteReader,
+    ByteWriter,
+    Deserializable,
+    DeserializationError,
+    Felt,
+    HashedStorageMapKey,
+    Serializable,
+};
+use crate::Hasher;
+use crate::asset::AssetVaultKey;
+
+#[derive(Debug, PartialEq, Eq, Clone, Copy, PartialOrd, Ord)]
+pub struct StorageMapKey(Word);
```

Comment:
We should soon have the ability to add `#[derive(WordWrapper)]` introduced in https://github.com/0xMiden/miden-base/pull/2108, and once that is merged we should add it.